pkg/aws/v1alpha3/awscluster/validate_awscluster.go

```diff
@@ -38,6 +38,61 @@ func NewValidator(config config.Config) (*Validator, error) {
 }
 
 func (v *Validator) Validate(request *admissionv1.AdmissionRequest) (bool, error) {
```

Comment:
This function was split into Create vs Update case because the namespace is only checked on creation.